<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-N42I4EB" failures="1" tests="2" name="com.hashTagMIS.Test.Admin.A2Employee.AdEmpTC" time="4.378" errors="0" timestamp="2024-07-20T11:46:36 IST" skipped="1">
  <testcase classname="com.hashTagMIS.Test.Admin.A2Employee.AdEmpTC" name="adminLogin" time="4.378">
    <failure message="The following asserts failed:
	element at index0Dont Matched expected [Mark] but found [],
	element at index3Dont Matched expected [Maintenance] but found [Care Support],
	element at index5Dont Matched expected [Care Support] but found [E-Commerce],
	element at index6Dont Matched expected [E-Commerce] but found [Maintenance],
	Lists differ at element [0]: Mark !=  expected [Mark] but found []" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
element at index0Dont Matched expected [Mark] but found [],
element at index3Dont Matched expected [Maintenance] but found [Care Support],
element at index5Dont Matched expected [Care Support] but found [E-Commerce],
element at index6Dont Matched expected [E-Commerce] but found [Maintenance],
Lists differ at element [0]: Mark !=  expected [Mark] but found []
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at com.hashTagMIS.Test.Admin.A2Employee.AdEmpTC.adminLogin(AdEmpTC.java:151)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- adminLogin -->
  <system-out>
    <![CDATA[Staff Added<====>Staff Added
<== Verify data in Dashboard ==>
<== Verify data in Emp edit Page ==>
<== Verify data in Emp Credentials ==>]]>
  </system-out>
  <testcase classname="com.hashTagMIS.Test.Admin.A2Employee.AdEmpTC" name="DeleteAllStaff">
    <skipped/>
  </testcase> <!-- DeleteAllStaff -->
  <system-out/>
</testsuite> <!-- com.hashTagMIS.Test.Admin.A2Employee.AdEmpTC -->
